app/Resources/views/page.html.twig

```diff
@@ -189,6 +192,12 @@
 {% endfor %}
 {% endblock %}
 
+{% block stylesheets deferred %}
+{% for stylesheet in stylesheets %}
+    <link href="{{ asset(stylesheet) }}" rel="stylesheet">
```

Comment:
This is valid in HTML5 https://html.spec.whatwg.org/multipage/links.html#body-ok